tests/tlm/test_properties.py

```diff
@@ -1,3 +1,204 @@
+import asyncio
+from typing import Any, Dict, List, Union
+
 import pytest
 
-# Empy file - no code to test yet, just so CI passes
+from cleanlab_studio.internal.constants import (
+    _VALID_TLM_MODELS,
+    _VALID_TLM_QUALITY_PRESETS,
+)
+from cleanlab_studio.studio.trustworthy_language_model import TLM
+from tests.tlm.test_get_trustworthiness_score import is_trustworthiness_score
+from tests.tlm.test_prompt import is_tlm_response
+
+_VALID_TLM_MODELS = ["gpt-4o-mini", "gpt-3.5-turbo-16k", "gpt-4", "gpt-4o", "claude-3-haiku"]
```

Comment:
Also, `_VALID_TLM_MODELS` is additionally imported above, so this var is overloaded

Reply:
Overloaded initially for testing with a single model but good catch, I will properly handle this